evm_loader/program/src/instruction.rs

```diff
@@ -95,11 +95,16 @@ pub enum EvmInstruction<'a> {
 
         /// Owner program account address
         owner: Pubkey,
+
+        /// Associated token address to create
+        token: Option<Pubkey>,
```

Comment:
In what cases is it not necessary to create a balance?

Reply:
Is `CreateAccountWithSeed` will only be used for creating holder token accounts?

Reply:
@anton-lisanin I think we do not have a case when we create an account without balance.

Reply:
@s-medvedev It is not a balance account. @otselnik creates a new token account and use it to store gas_limit amount of tokens from balance account.